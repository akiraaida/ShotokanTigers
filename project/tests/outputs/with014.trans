10 Loaded Guy           00000 00000.00 S 
01 Loaded Guy           12353 00500.00   
01 Loaded Guy           12353 00000.05   
01 Loaded Guy           12354 00500.00   
01 Loaded Guy           12354 00000.10   
00                      00000 00000.00   
