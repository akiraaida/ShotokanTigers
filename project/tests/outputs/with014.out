Please enter a command: 
Please enter your session type: 
Please enter a login name: 
Please enter a command: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12353):
0.00
Daily Withdrawals Remaining (12353):
0.00
Please enter a command: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12354):
0.00
Daily Withdrawals Remaining (12354):
0.00
Please enter a command: 
